home/index.html

```diff
@@ -105,6 +101,11 @@ <h2 class="section" id="overview">Features</h2>
 <p>
   This build file also includes a directive to upload your artifacts to Bintray automatically.
 </p>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Fast builds</h2>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Intuitive plug-in architecture</li></h2>
+  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Half the size of Gradle</h2>
```

Comment:
Hi Cedric,

I made the suggested changes, I think.

You have more experience but i've read that people don't change their ways
unless the new alternative is 9 times better than the old, hence my concern
for highlighting the advantages.

Best,
Juan

On Sun, Jan 10, 2016 at 2:09 PM, Juan Liska <jp@guatec.com> wrote:

> Yeah, i thought you might not like that.  It's one of the things I like
> about Kobalt, esp considering it already includes the kotlin compiler.
>
> But I'll remove it.
>
> On Sun, Jan 10, 2016 at 1:56 PM, Cedric Beust <notifications@github.com>
> wrote:
>
>> In home/index.html
>> <https://github.com/cbeust/kobalt-doc/pull/9#discussion_r49280567>:
>>
>> > @@ -105,6 +101,11 @@ <h2 class="section" id="overview">Features</h2>
>> >  <p>
>> >    This build file also includes a directive to upload your artifacts to Bintray automatically.
>> >  </p>
>> > +  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Fast builds</h2>
>> > +  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Intuitive plug-in architecture</li></h2>
>> > +  <h2 class="section" id="syntax" indent="1"><span class="glyphicon glyphicon-check"></span> Half the size of Gradle</h2>
>>
>> I don't think we should say it's half the size of Gradle: users won't
>> care about this and who knows, Kobalt might end up being pretty big too...
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/cbeust/kobalt-doc/pull/9/files#r49280567>.
>>
>
>